### PR TITLE
v1.0.0

### PR DESCRIPTION
**Pull request by ravenjohn**
Changes:
- Instead of loading anytv-i18n by itself, just let the user do it to avoid loading the library multiple times
- added `CHANGELOG.md`

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+v1.0.0
+-----
+
+* **Breaking Change:** Will not configure or load the `anytv-i18n` module anymore.
```
Comment (pandafox) [inline CHANGELOG.md:4]: I guess this is to make sure i18n module can be re-used? Where did you observe where i18n is loaded multiple times?

Comment (ravenjohn) [reply CHANGELOG.md:4]: in unlink-request. it uses both anytv-mailer and anytv-templater which resulted to loading the locales twice.

Comment (pandafox) [reply CHANGELOG.md:4]: Right, I guess we don't need the mailer now we're not sending email anymore?

**Review by pandafox — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ravenjohn) [reply CHANGELOG.md:4]: it still uses anytv-mailer, the part where we auto denied an unlink request and we tell them why.

**Review by wilbertverayin — APPROVED** [review]

**Pull request merged**